src/app/pages/sharing/iscsi/extent/extent-form/extent-form.component.ts

```diff
@@ -38,23 +44,23 @@ export class ExtentFormComponent {
     },
     {
       type: 'select',
-      name: 'iscsi_target_extent_type',
+      name: 'type',
       placeholder: T('Extent type'),
       tooltip: T('Select from <i>File</i> or <i>Device</i>.'),
```

Comment:
`` <i>File</i> shares the contents of an individual file. <i>Device</i> shares an entire device. ``